downstream/assemblies/dev-guide/assembly-tools-components.adoc

```diff
@@ -0,0 +1,35 @@
+
+
+ifdef::context[:parent-context: {context}]
+
+
+
+[id=" stools"]
+=  Tools and components
+
+
+:context: tools
+
+
+[role="_abstract"]
+Learn more about the {PlatformName} tools and components you will use in creating {ExecEnvName}.
+
+// include::builder/con-about-builder.adoc[leveloffset=+1]
+
+include::navigator/con-about-ansible-navigator.adoc[leveloffset=+1]
+
+include::automation-hub/con-about-automation-hub.adoc[leveloffset=+1]
```

Comment:
There's a problem with /automation-hub/con-about-automation-hub.adoc: the content under = About Automation Hub is indented, so it renders as a blockquote in the built docs.